Tests/ApodiniTests/RelationshipTests/Infrastructure/RelationshipExporter.swift

```diff
@@ -5,14 +5,28 @@
 import XCTest
 import XCTApodini
 @testable import Apodini
+@testable import ApodiniREST
 
 class RelationshipExporter: MockExporter<String> {
-    var endpoints: [(AnyEndpoint, AnyRelationshipEndpoint)] = []
+    // swiftlint:disable:next large_tuple
+    var endpoints: [(AnyEndpoint, AnyRelationshipEndpoint, (String, [Any??], Application) throws -> EnrichedContent)] = []
```

Comment:
Well I think the SwiftLint warning has its place here, I don't understand shit 😅 maybe a sweet struct would do the job?

Reply:
You got me there🙈